tests/components/ct-react-vite/package.json

```diff
@@ -19,9 +19,5 @@
     "@vitejs/plugin-react": "^3.1.0",
     "typescript": "^4.5.4",
     "vite": "^4.2.1"
-  },
-  "@standaloneDevDependencies": {
```

Comment:
what was this?

Reply:
This was a hack to give user a hint how to run it in a standalone (non monorepo) mode.